Added Nation-Wide and State-Wide Holidays in India as per Gregorian Calendar

DrunkenCloud opened the pull request:
Many holidays follow their own calendars like the Hindu or Islam calendar. States as well have more authority to choose holidays so nation wide holidays are limited.

Issue: https://github.com/commenthol/date-holidays/issues/527

DrunkenCloud commented:
Added a fix. Appologies

ranjith-gampa commented:
Nice to see to support for India added. Great work! I wonder how could we add support for dynamic holidays like festivals in India. I feel the directory structure should look like this 

country 
 -> {name} 
       -> constant_holidays
       -> dynamic_holidays
       
dynamic_holidays might need to be updated start of every year.

DrunkenCloud commented:
Yeah that should work, I suppose the parsing will change, not that much so yeah. Currently its like IN.yaml, and if im not wrong, your proposing

countries/
    IN/
     |---- constant.yaml // constant holidays
     |---- dynamic.yaml // dynamic holidays

for each country? keeping the current file structure same. There are some opensource tools that give festivities dates, so I suppose we will be using those dates for each year.

DrunkenCloud commented:
I just noticed that languages support in the file was wrong, fixed that. There are some spelling mistakes in the tamil names, I did that on purpose due to some unicode problems, I dont understand why, it doesnt render properly, so I rewrote it to work (My mother tongue is Tamil)

hrishabh-kumar commented:
Glad to see the support for Indian calendar, although I see `Good Friday` is missing from the list of holidays.

```
easter -2:
  name:
    en: Good Friday
    hi: गुड फ्राइडे
    ta: புனித வெள்ளி
```

DrunkenCloud commented:
Thanks for noticing. Added it!

commenthol commented:
Hi @DrunkenCloud,
The "regions:" property is not known. Can you please check the specification.md?

DrunkenCloud commented:
On it. Im currently changing the format.

DrunkenCloud commented:
working on it
